src/guacd-docker/bin/build-all.sh

```diff
@@ -95,11 +95,29 @@ install_from_git() {
 
 }
 
+#
+# Determine any option overrides to guarantee successful build
+#
+
+export BUILD_ARCHITECTURE="$(arch)" # Determine architechutre building on
+echo "Build architecture: $BUILD_ARCHITECTURE"
+
+case $BUILD_ARCHITECTURE in
+    armv6l|armv7l|aarch64)
+        echo "Using ARM overrides"
```

Comment:
These echo statements look useful for debugging this change, but not so useful for general usage. If you look at above logging examples like `"Building $REPO_DIR @ $VERSION ..."`, it includes useful information that varies from build to build, where as "Using ARM overrides" is going to appear verbatim in the build output of every single ARM build ever, and also doesn't add any additional information that the output on line `103` isn't already telling you.

I'd just remove these last 2 log lines entirely.